concise-mid-tag.cddl

```diff
@@ -136,8 +136,14 @@ tagged-ueid-type = #6.48000(ueid-type)
 impl-id-type = bytes .size 32
 tagged-impl-id-type = #6.47115(impl-id-type)
 
+$measured-element-type-choice /= tstr
+
+measurement-map = [
```

Comment:
Is the array construct "[" required here or it should be { 
??

Reply:
I guess it's the most efficient way to model a pair with one optional element

Reply:
I would prefer to name it as :
measured-identifier-type-choice

Reply:
Also the name measurement-map on the top level does not seem right to me, if you intend to define it as an array?

Reply:
> Also the name measurement-map on the top level does not seem right to me, if you intend to define it as an array?

I haven't changed the top-level type name in order to show the (smooth) transition, but I'm aware this doesn't fit with our TYPOGRAPHY.md.   I am happy to change it to something else (e.g., `measurement-record`)

Reply:
> I would prefer to name it as :
> measured-identifier-type-choice

this is not a "measured identifier", it's the identifier of a measured "thing", which for lack of better alternatives I decided to call "element"

Reply:
True and Thanks @thomas-fossati 
I just checked CDDL spec, and what we are doing here calling it a map and defining an array would very much confuse the reader: (like me especially :) )

Reply:
We discussed an alternative approach:
Option 2:

measurement-map = {
? comid.identifier-for-meas => $measured-element-type-choice,
  comid.meas-value => measurement-values-map
}


Let's conclude later in the afternoon today, what the team thinks as a better option and close it then!

@thomas-fossati Thank you for your valuable points during discussion. Appreciate it!

Reply:
Let's go with the map approach

Reply:
done in 509499e